applications/ContactStructuralMechanicsApplication/tests/test_double_curvature_integration.py

```diff
@@ -134,40 +135,40 @@ def _moving_nodes_tests(self, input_filename, num_nodes):
             for node in self.main_model_part.GetSubModelPart("GroupNegativeY").Nodes:
                 node.Y -= delta_disp
             del(node)
-            
+
             #print("Solution obtained")
             tolerance = 5.0e-5
             for cond in self.contact_model_part.Conditions:
                 if cond.Is(KratosMultiphysics.SLAVE):
                     area = self.exact_integration.TestGetExactAreaIntegration(self.contact_model_part, cond)
-                    condition_area = cond.GetArea() 
+                    condition_area = cond.GetArea()
                     check_value = abs((area - condition_area)/condition_area)
                     if (check_value >  tolerance):
                         print(cond.Id,"\t",area,"\t", condition_area,"\t", self.__sci_str(check_value))
                     else:
                         self.assertLess(check_value, tolerance)
-                    
+
     def test_double_curvature_integration_triangle(self):
         input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_double_curvature_integration_triangle"
-        
+
         # These conditions are in the border, and can not be integrated 100% accurate
         list_of_border_cond = [1262,1263,1264,1265,1269,1270,1273,1275,1278,1282,1284,1285,1286,1288,1290,1291,1292,1294,1295,1297,1298,1302,1303,1305,1306,1307,1310,1313,1314,1318,1319,1320,1323,1325,1327,1328,1329,1331,1336,1337,1338,1340,1341,1342,1343,1344,1346,1347,1348,1349,1350,1353,1355,1357,1359,1360,1366,1367,1368,1369,1370,1377,1378,1379,1381,1382,1384,1385,1387,1393,1394,1395,1399,1400,1406,1410,1411,1412,1414,1415,1418,1419,1420,1424,1427,1429,1431,1436,1438,1444,1446,1447,1448,1449,1459,1462,1463,1465,1467,1468,1474,1477,1479,1485,1491,1493,1507,1515,1517,1531,1537,1539,1547,1549,1553,1563,1569,1575,1623,1640,1644,1654,1656,1663,1667,1675,1685,1687,1693,1697,1703,1707,1713,1715,1717,1719,1721,1723,1725]
-        
+
         self._double_curvature_tests(input_filename, 3, list_of_border_cond)
-        
+
     def test_double_curvature_integration_quad(self):
         input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_double_curvature_integration_quadrilateral"
-        
+
         # These conditions are in the border, and can not be integrated 100% accurate
```

Comment:
why so? (not a blocker, just a curiosity)

Reply:
Imagine that you are projecting from a certain distance (the set of conditions are not touching), the conditions in the border will lack some area because of that